### DISCUSSION TITLE
can someone tell me why my extension doesn't work

### DISCUSSION THREAD

**Discussion by lego7set** [TurboWarp/extensions#1237] [Q&A]
im making a goto extension and i have no idea how to do what I'm going to do, or if it's even possible. 

[gotoPm.txt](https://github.com/TurboWarp/extensions/files/13924891/gotoPm.txt) <--- link to file (ignore pm)

can someone tell me how i can get the second block working or whats wrong with my code? i have no idea how to make that happen. how do i get the thread to start executing the block that created the label, and execute the blocks after?

**Comment by GarboMuffin** [comment]
maybe you can make that work in the interpreter but that will never work in the compiler

**Reply by GarboMuffin** [reply]
it looks like it does work (for some definition of work) if you disable the compiler
